src/main/java/org/terasology/ligthandshadow/componentsystem/controllers/ClientStatisticsSystem.java

```diff
@@ -0,0 +1,117 @@
+    public void initialise() {
+        statisticsScreen = nuiManager.createScreen("LightAndShadow:statisticsScreen", DeathScreen.class);
+        clientGameOverSystem.addGoalScore(statisticsScreen, "spadesGoalScore");
+        clientGameOverSystem.addGoalScore(statisticsScreen, "heartsGoalScore");
+        clientGameOverSystem.addTeamScore(statisticsScreen, "heartsTeamScore", redScore);
+        clientGameOverSystem.addTeamScore(statisticsScreen, "spadesTeamScore", blackScore);
+
+    }
+    @ReceiveEvent
+    public void onScoreUpdate(ScoreUpdateFromServerEvent event, EntityRef entity) {
+        String team = event.team;
+        if (team.equals(LASUtils.RED_TEAM)) {
+            redScore = event.score;
+        }
+        if (team.equals(LASUtils.BLACK_TEAM)) {
+            blackScore = event.score;
+        }
+    }
+
+    @ReceiveEvent(priority = EventPriority.PRIORITY_CRITICAL)
+    public void onTab(TabButton event, EntityRef entity) {
+        if (event.isDown()) {
+            if (localPlayer.getClientEntity().equals(entity) && !isOpen) {
+                if (!isExpired) {
+                    entity.send(new ExpireNotificationEvent(NOTIFICATION_ID));
+                    isExpired = true;
+                }
+                clientGameOverSystem.addTeamScore(statisticsScreen, "spadesTeamScore", blackScore);
+                clientGameOverSystem.addTeamScore(statisticsScreen, "heartsTeamScore", redScore);
+                clientGameOverSystem.addPlayerStatisticsInfo(statisticsScreen);
+                nuiManager.pushScreen("LightAndShadow:statisticsScreen");
+                isOpen = true;
+                event.consume();
+            }
+        } else {
+            isOpen = false;
+            nuiManager.closeScreen("LightAndShadow:statisticsScreen");
+        }
+    }
+
+    private String getActivationKey(SimpleUri button) {
+        return inputSystem.getInputsForBindButton(button).stream()
+                .findFirst()
+                .map(Input::getDisplayName)
+                .map(key -> {
+                    if (key.length() == 1) {
+                        // print the key in yellow within a circle
+                        int off = key.charAt(0) - 'A';
+                        char code = (char) (EnclosedAlphanumerics.CIRCLED_LATIN_CAPITAL_LETTER_A + off);
+                        return String.valueOf(code);
+                    } else {
+                        return key;
+                    }
+                })
+                .map(key -> FontColor.getColored(key, Color.yellow))
+                .orElse(FontColor.getColored("n/a", Color.red));
+    }
+
+    @ReceiveEvent
+    public void onLocalPlayerInitialized(LocalPlayerInitializedEvent event, EntityRef entity) {
+            Notification notification = new Notification(NOTIFICATION_ID,
+                    "The Numbers Game",
+                    "Press " + getActivationKey(new SimpleUri("LightAndShadow:statistics")) + " to see statistics screen",
```

Comment:
Either go with 
```suggestion
                    "Press " + getActivationKey(new SimpleUri("LightAndShadow:statistics")) + " to see statistics",
```
as I proposed originally, or with
```suggestion
                    "Press " + getActivationKey(new SimpleUri("LightAndShadow:statistics")) + " to see the statistics screen",
```

Currently, it's a bit weird English :sweat_smile: 
I would prefer the first as it's shorter.